app/assets/javascripts/miq_application.js

```diff
@@ -841,7 +841,10 @@ function miqBuildCalendar() {
       element.datepicker();
     }
 
-    if (ManageIQ.calendar.calDateFrom) {
+    var start_date = element.attr('data_date_start');
+    if (start_date) {
```

Comment:
@ailisp for sure heed the warnings above and change `start_date` to `startDate` . . . thx, Dan